R/utils.R

```diff
@@ -49,11 +49,16 @@ generate_random_id <- function(prefix, id_length = 20) {
 #'
 #' This throws warning if there are parameters not supported by semantic.
 #'
-#' @param args list with extra arguments
+#' @param args list or vector with extra arguments
 check_extra_arguments <- function(args) {
-  to_wrn <- paste0(as.character(names(args)), collapse = ',')
-if (nchar(to_wrn) > 1)
-  warning(glue::glue("arguments: `{to_wrn}` not supported yet in semantic version"))
+  if (class(args) == "list")
+    to_wrn <- paste0(as.character(names(args)), collapse = ',')
+  else if (class(args) == "character")
+    to_wrn <- paste0(args, collapse = ',')
+  else
+    stop("Wrong input type!")
+  if (nchar(to_wrn) > 1)
+    warning(glue::glue("arguments: `{to_wrn}` not supported yet in semantic version"))
```

Comment:
"semantic version" can be ambiguous, let's refer to the full package name "arguments (...) not supported yet in this shiny.semantic version"